src/Compilers/CSharp/Portable/CSharpExtensions.cs

```diff
@@ -1220,6 +1220,15 @@ public static ISymbol GetDeclaredSymbol(this SemanticModel semanticModel, Declar
         }
 
         /// <summary>
+        /// Given a declaration pattern syntax, get the corresponding symbol.
+        /// </summary>
+        public static ISymbol GetDeclaredSymbol(this SemanticModel semanticModel, LetStatementSyntax declarationSyntax, CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
This is the only place where the comment is incorrect. I'll adjust it in the next change.